py3dtilers/TilesetReader/tile_hierarchy.py

```diff
@@ -0,0 +1,74 @@
+from ..Common import GeometryNode, GeometryTree, ObjectsToTile
+
+
+class TileHierarchy(GeometryTree):
```

Comment:
Still semantics: if we follow the Tree/Node terminology this is a TileSetTree. No ?